RecoPixelVertexing/PixelTriplets/plugins/CAHitQuadrupletGeneratorGPU.cc

```diff
@@ -11,8 +11,8 @@
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 #include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
 #include "FWCore/Utilities/interface/isFinite.h"
+#include "RecoPixelVertexing/PixelTriplets/interface/CellularAutomaton.h"
 #include "RecoPixelVertexing/PixelTriplets/interface/ThirdHitPredictionFromCircle.h"
-#include "RecoPixelVertexing/PixelTriplets/src/CellularAutomaton.h"
```

Comment:
@makortel , the internal CA headers were already being included outside of the library itself, so they should live in `interface`.

Reply:
In patatrack, yes, but not in CMSSW master.